nanosynth.rb

```diff
@@ -28,11 +28,13 @@
 ###
 ### http://www.joelstrait.com/nanosynth_create_sound_with_ruby/
 
-gem 'wavefile', '=0.7.0'
+gem 'wavefile'
```

Comment:
I intentionally indicated a specific version here, to avoid issues if the user has a different version of the gem installed. Previous versions of the gem have incompatible differences in the API, and it could cause confusing issues for the user. Although I could add a `Gemfile` and require the user to use bundler to run the script, I wanted to keep this as simple as possible so I went with this approach. As a result, I don't want to make this change.